### PR TITLE
LTP: syscalls: updating list of known issues

### PR DESCRIPTION
**Pull request by nareshkamboju**

### CODE REVIEW

File: ltp-production.yaml [hunk 1]
```diff
@@ -1792,3 +1792,15 @@ projects:
...
+  - environments: *environments_all
+    notes: >
+      LTP syscalls recvmmsg01 fails on compat mode and 32-bit mode.
+      recvmmsg01[421]: segfault at error 4 in libc.so.6
+      comm=\"recvmmsg01\" exe=\"/opt/ltp/testcases/bin/recvmmsg01\" sig=11 res=1
+      tst_test.c:1618: TBROK: Test killed by SIGSEGV!
+    projects: *projects_all
+    test_names:
+    - ltp-syscalls/recvmmsg01
+    url: https://bugs.linaro.org/show_bug.cgi?id=5980
```
Comment (roxell) [inline ltp-production.yaml:1804]: is this reported upstream?
if yes, can we add the url to that report too?

File: ltp-production.yaml [hunk 2]
```diff
@@ -1778,3 +1778,17 @@ projects:
...
+      LTP syscalls mq_timedreceive01 and mq_timedsend01 fails on compat mode and 32-bit mode.
+      mq_timedreceive[733]: segfault at f7f8d004 ip 00000000f7c93b5f sp 00000000ff92e9a0 error 4 in libc.so.6
+      tst_test.c:1618: TBROK: Test killed by SIGSEGV!
+      mq_timedsend01.c:210: TPASS: mq_timedsend() failed expectedly: EFAULT (14)
+      tst_test.c:1618: TBROK: Test killed by SIGSEGV!
+    projects: *projects_all
+    test_names:
+    - ltp-syscalls/mq_timedreceive01
+    - ltp-syscalls/mq_timedsend01
+    url: https://bugs.linaro.org/show_bug.cgi?id=5979
```
Comment (roxell) [inline ltp-production.yaml:1792]: is this reported upstream?
if so, can that url be added too.

File: ltp-production.yaml [hunk 3]
```diff
@@ -1767,3 +1767,14 @@ projects:
...
     intermittent: true
+  - environments: *environments_all
+    notes: >
+      sigtimedwait01 fails on compat mode 64-bit kernel booting on 32-bit userspace
+      sigtimedwait01[566]: segfault at 5 ip 00000000f7c3b58f sp 00000000ffe587d0 error 4 in libc.so.6
+      tst_test.c:1618: TBROK: Test killed by SIGSEGV!
+    projects: *projects_all
+    test_names:
+    - ltp-syscalls/sigtimedwait01
+    url: https://bugs.linaro.org/show_bug.cgi?id=5978
```
Comment (roxell) [inline ltp-production.yaml:1778]: is this reported upstream?
if so, can that url be added too.

File: ltp-production.yaml [hunk 4]
```diff
@@ -1757,3 +1757,13 @@ projects:
...
     active: true
     intermittent: true
+  - environments: *environments_all
+    notes: >
+      shmget02 fails on compat mode 64-bit kernel booting on 32-bit userspace
+      shmget02.c:95: TFAIL: shmget(1644199826, 4278190080, 1536) expected EINVAL: ENOMEM (12)
+    projects: *projects_all
+    test_names:
+    - ltp-syscalls/shmget02
+    url: https://bugs.linaro.org/show_bug.cgi?id=5977
```
Comment (roxell) [inline ltp-production.yaml:1767]: reported upstream? url?

File: ltp-production.yaml [hunk 5]
```diff
@@ -1195,13 +1195,10 @@ projects:
     url: https://bugs.linaro.org/show_bug.cgi?id=5640
     active: true
     intermittent: true
-  - environments:
-    - juno-r2
-    - juno-r2-kasan
-    - juno-r2-compat
-    - juno-64k_page_size
+  - environments: *environments_all
```
Comment (roxell) [inline ltp-production.yaml:1198]: when did these start to fail on all devices?
And no one have replied back to your reports upstream?

File: ltp-production.yaml [hunk 6]
```diff
@@ -1723,10 +1723,11 @@ projects:
     url: https://bugs.linaro.org/show_bug.cgi?id=5806
     active: true
     intermittent: true
-  - environments: *environments_qemu
+  - environments: *environments_all
```
Comment (roxell) [inline ltp-production.yaml:1712]: isn't we disabling this from a bit too much now?
if it only fails on qemu and compat?

File: ltp-production.yaml [hunk 7]
```diff
@@ -1758,3 +1758,15 @@ projects:
...
+    notes: >
+      io_uring01 and io_uring02 fails on compat mode 64-bit kernel booting on 32-bit userspace
+      io_uring01.c:89: TBROK: mmap((nil),388,3,32769,3,0) failed: ENOMEM (12)
+      io_uring02.c:213: TBROK: mmap((nil),18752,3,32769,5,0) failed: ENOMEM (12)
+    projects: *projects_all
+    test_names:
+    - ltp-syscalls/io_uring01
+    - ltp-syscalls/io_uring02
+    url: https://bugs.linaro.org/show_bug.cgi?id=5976
```
Comment (roxell) [inline ltp-production.yaml:1757]: reported upstream? URL?

File: ltp-production.yaml [hunk 8]
```diff
@@ -1758,3 +1758,15 @@ projects:
     url: https://bugs.linaro.org/show_bug.cgi?id=5953
     active: true
     intermittent: true
+  - environments: *environments_all
```
Comment (roxell) [inline ltp-production.yaml:1748]: if its only failing with compat mode, why disable for all environments?

File: ltp-production.yaml (hunk 1 above)
Comment (nareshkamboju) [reply ltp-production.yaml:1804]: Reported upstream and link have been updated.

File: ltp-production.yaml (hunk 2 above)
Comment (nareshkamboju) [reply ltp-production.yaml:1792]: Reported upstream and link have been updated.

File: ltp-production.yaml (hunk 3 above)
Comment (nareshkamboju) [reply ltp-production.yaml:1778]: Reported upstream and link have been updated.

File: ltp-production.yaml (hunk 5 above)
Comment (nareshkamboju) [reply ltp-production.yaml:1198]: Test case is failing on one of those environments is expected to fail on future devices / environments. It is not easy to maintain the list of environments update manually.

Reports are getting noise due to this test case added to limited number of environments as a known issue.

File: ltp-production.yaml (hunk 6 above)
Comment (nareshkamboju) [reply ltp-production.yaml:1712]: Test case is failing on one of those environments is expected to fail on future devices / environments. It is not easy to maintain the list of environments update manually.

Reports are getting noise due to this test case added to limited number of environments as a known issue.

File: ltp-production.yaml (hunk 7 above)
Comment (nareshkamboju) [reply ltp-production.yaml:1757]: Yet to report few failures on LTP mailing list and LKML.

File: ltp-production.yaml (hunk 8 above)
Comment (nareshkamboju) [reply ltp-production.yaml:1748]: same as above reason.

File: ltp-production.yaml (hunk 4 above)
Comment (nareshkamboju) [reply ltp-production.yaml:1767]: Reported internally and did not get any response.

**Comment by roxell** [comment]
please report the last few compat kernel testing upstream and ask if its meant to work or not. Just so we have it tracked.

File: ltp-production.yaml (hunk 7 above)
Comment (nareshkamboju) [reply ltp-production.yaml:1757]: Reported upstream.

**Comment by nareshkamboju** [comment]
Folliowing list of bugs have been created, updated and reported.
 - https://bugs.linaro.org/show_bug.cgi?id=5980
 - https://bugs.linaro.org/show_bug.cgi?id=5979
 - https://bugs.linaro.org/show_bug.cgi?id=5978
 - https://bugs.linaro.org/show_bug.cgi?id=5986
 - https://bugs.linaro.org/show_bug.cgi?id=5977
 - https://bugs.linaro.org/show_bug.cgi?id=5976
 - https://bugs.linaro.org/show_bug.cgi?id=5953
 - https://bugs.linaro.org/show_bug.cgi?id=5952
 - https://bugs.linaro.org/show_bug.cgi?id=5951

File: ltp-production.yaml (hunk 4 above)
Comment (nareshkamboju) [reply ltp-production.yaml:1767]: Reported upstream.

**Pull request merged**